ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarDemoController.swift

```diff
@@ -196,6 +197,21 @@ class AvatarDemoController: DemoTableViewController {
         }
     }
 
+    private var isShowingActivity: Bool = false {
+        didSet {
+            if oldValue != isShowingActivity {
+                var activityStyle: MSFAvatarActivityStyle
+                var isEven: Bool
+                for index in 0 ..< allDemoAvatarsCombined.count {
+                    isEven = index % 2 == 0
+                    activityStyle = isEven ? .circle : .square
+                    allDemoAvatarsCombined[index].state.activityStyle = isShowingActivity ? activityStyle : .none
+                    allDemoAvatarsCombined[index].state.activityImage = isEven ? nil : UIImage(named: "excelIcon")
```

Comment:
excel icon is only for the square style right? can we add another image for the circular one?

Reply:
grab some official emoji https://github.com/microsoft/fluentui-emoji